src/client/util/http.h

```diff
@@ -60,6 +60,25 @@ enum struct Timeout : std::uint8_t {
   Receive = 30,
 };
 
+/// @enum LengthLimit
+/// @brief Arbitrary upper limits and specified lower limits on HTTP download size
+enum LengthLimit : std::uint32_t
```

Comment:
This is unique to class HTTP and should be scoped as such. Also s/LengthLimit/Length/ because limit is implied.

Reply:
I left the scope off for implicit conversion. Will add the scope, and explicit casts in the impl.

Reply:
Ok, I'll move the enum into the HTTP class, and rename it.